destination/destination_test.go

```diff
@@ -93,29 +94,49 @@ func TestDestination_Write(t *testing.T) {
 		ctrl := gomock.NewController(t)
 		ctx := context.Background()
 
-		record := sdk.Record{
+		metadata := sdk.Metadata{}
+		metadata.SetCreatedAt(time.Now())
+
+		records := make([]sdk.Record, 2)
+		records[0] = sdk.Record{
 			Position: sdk.Position("1.0"),
-			Metadata: map[string]string{
-				"action": "insert",
-			},
+			Metadata: metadata,
 			Key: sdk.StructuredData{
 				"id": 1,
 			},
-			Payload: sdk.StructuredData{
-				"id":   1,
-				"name": "Void",
+			Payload: sdk.Change{
+				After: sdk.StructuredData{
+					"id":   1,
+					"name": "John",
+				},
+			},
+		}
+		records[1] = sdk.Record{
+			Position: sdk.Position("1.0"),
```

Comment:
sdk.Position("1.0")=> please change to your position

Reply:
Changed.